conf.d/z.fish

```diff
@@ -14,50 +14,25 @@ if test ! -e "$Z_DATA"
     touch "$Z_DATA"
 end
 
-if test -z "$Z_CMD"
-    set -U Z_CMD z
-end
-
-set -U ZO_CMD "$Z_CMD"o
-
-if test ! -z $Z_CMD
-    function $Z_CMD -d "jump around"
```

Comment:
I think `jump around` and `open target dir` (line 30 of this file), were the descriptions that were used by fish for these functions. I kept the one that was in `__z.fish`, but if you want we can use these instead.